mne/source_estimate.py

```diff
@@ -452,8 +452,10 @@ class _BaseSourceEstimate(ToDataFrameMixin, TimeMixin):
         a tuple with two arrays: "kernel" shape (n_vertices, n_sensors) and
         "sens_data" shape (n_sensors, n_times). In this case, the source
         space data corresponds to "numpy.dot(kernel, sens_data)".
-    vertices : array | list of array
-        Vertex numbers corresponding to the data.
+    vertices : array | list of array, shape (2,)
+        Vertex numbers corresponding to the data. If a list, the first element
+        of the list contains vertices of left hemisphere and the
+        second element contains vertices of right hemisphere.
```

Comment:
as a mixed or volume source space can inherit from this base class I don't think this is correct

Reply:
If it's a list it will necessarily contain two elements, no? Or could be either a list with an array in it or simply an array?